native/src/components/MapView.tsx

```diff
@@ -76,7 +76,6 @@ const MapView = React.forwardRef(
     const coordinates = selectedFeature?.geometry.coordinates
     const defaultSettings: CameraSettings = {
       zoomLevel: coordinates ? detailZoom : defaultViewportConfig.zoom,
-      centerCoordinate: coordinates,
```

Comment:
why is this removed?

Reply:
we have the `setCamera` in the `useEffect` of the PoisPage (with the padding for the bottomActionSheet), when a poi is selected. so this is centering once and then the effect is called so its actually unneeded